VeganVirus/VeganVirus/FruitThrow.cpp

```diff
@@ -0,0 +1,100 @@
+#include "FruitThrow.h"
+#include <stdlib.h>
+#include <math.h>
+
+const std::vector<const wchar_t*> FruitThrow::fruitImages = {
+	L"fruit0.png",
+	L"fruit1.png",
+	L"fruit2.png",
+	L"fruit3.png",
+	L"fruit4.png",
+	L"fruit5.png",
+	L"fruit6.png",
+	L"fruit7.png",
+	L"fruit8.png",
+};
+
+FruitThrow::FruitThrow(Draw* draw, MouseManager& mouseManager) : _draw(draw), _mouseManager(mouseManager)
+{
+	// randomize fields
+	_r += rand() % RADIUS_RANGE;
+
+	const wchar_t* path = FruitThrow::fruitImages[rand() % FruitThrow::fruitImages.size()];
+	this->_image = Draw::resizedBitmap(path, _r * 2, _r * 2);
+
```

Comment:
I think you need to add comment  it have a lot of calculations  and also i think magic number.

Reply:
the comment it for lines 26 to 33 consider add comment like // this calculations are for ....

Reply:
I think it's obvious that this lines are randomizing the new fruit's state. There's also a comment for it at the beginning of the function.
I'll fix the magic numbers.